extensions/wikia/AdEngine/js/ml/ctp/queenDesktopDataSource.js

```diff
@@ -0,0 +1,8 @@
+/*global define*/
+define('ext.wikia.adEngine.ml.ctp.queenDesktopDataSource', [
```

Comment:
Can we reverse it? According to AC:

> it can use the same data source for now (ctpDesktopDataSource.js) - rename it to queenOfHearts.js and alias ctpDesktop

Let queen to be the main source